exercises/concept/calculator-conundrum/.meta/src/reference/java/CalculatorConundrum.java

```diff
@@ -0,0 +1,30 @@
+public class CalculatorConundrum {
+    private final String answer;
+
+    public CalculatorConundrum(int operand1, int operand2, String operation) {
+        int result;
+        try {
+            if (operation.equals("+")) {
+                result = SimpleOperation.addition(operand1, operand2);
+            } else if (operation.equals("*")) {
+                result = SimpleOperation.multiplication(operand1, operand2);
+            } else if (operation.equals("/")) {
+                result = SimpleOperation.division(operand1, operand2);
+            } else if (operation.equals("")) {
+                throw new IllegalArgumentException("Operation cannot be empty");
+            } else if (operation == null) {
+                throw new NullPointerException("Operation cannot be null.");
+            } else {
+                throw new IllegalArgumentException(String.format("%s operation does not exist", operation));
+            }
+        } catch (ArithmeticException e){
+            throw new ArithmeticException(e.getMessage());
+        }
```

Comment:
As far as I can tell, there is no purpose to this try-catch block if it catches the thrown ArithmeticException only to re-throw the same exception type with the same message.

As the goal of the exercise is to teach exception handling, it would make more sense for them to catch the thrown ArithmeticException and then throw their own custom exception with a particular error message to verify that they did in fact catch the ArithmeticException and handle it.